README.md

```diff
@@ -37,6 +37,9 @@ requestInfoDefaultTitles:
 # *OPTIONAL* Label to be added to Issues and Pull Requests with insufficient information given
 requestInfoLabelToAdd: needs-more-info
 
+# *OPTIONAL* Require the issue to match an issue template you have in your repository. 
```

Comment:
Do you think maybe this could be better rephrased to explain what it's actually checking?